data/CTSI/data.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Cartesi Token",
+  "symbol": "CTSI",
+  "decimals": 18,
+  "description": "Cartesi - The Blockchain OS is a decentralized Layer-2 infrastructure that supports Linux and mainstream software components. For the first time, developers can code scalable smart contracts with rich software tools, libraries, and the services they’re used to, bridging the gap between mainstream software and blockchain.
```

Comment:
Last thing here -- make sure not to use newlines in the description, instead use `\n` where you want newlines to appear. But probably better to just not use newlines at all.